packages/frontend/core/src/modules/editor-setting/impls/doc-create-middleware.ts

```
@@ -28,23 +29,42 @@ export class EditorSettingDocCreateMiddleware
   extends Service
   implements DocCreateMiddleware
 {
+  private readonly allDocTitles$: LiveData<{ id: string; title: string }[]>;
```

ESLint

[error] Finnish notation should not be used here.

(rxjs/finnish)